### PR TITLE
Add more details about our deployments as well as a load testing script

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

File: tools/load-testing-before-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Prerequisites:
+# * installation steps in loadtest/README.md have been followed.
+# * using a connection capable a sustaining 50 MBits to the servers. A fiber connection is a must.
```
Comment (canova) [inline tools/load-testing-before-deployment.sh:5]: Is this 50 Mbits of upload or download?

File: tools/load-testing-before-deployment.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+echo "This script will run a large number of requests on both staging and dev environments."
+printf "Are you sure?"
+# shellcheck disable=SC2034
+read -r enter
+
+### And now, let's move on with the actual tasks.
+
+# A utility function to run the tests. Note that using () we spawn a subshell so
+# that directory changes are undone after the function returns.
+# This function needs to be run with the API_ENDPOINT env variable to designaet
```
Comment (canova) [inline tools/load-testing-before-deployment.sh:33]: nit:
```suggestion
# This function needs to be run with the API_ENDPOINT env variable to designate
```

**Review by canova — APPROVED** [review]
Looks good to me!

File: tools/load-testing-before-deployment.sh (hunk 1 above)
Comment (julienw) [reply tools/load-testing-before-deployment.sh:5]: upload :-)

Comment (julienw) [reply tools/load-testing-before-deployment.sh]: I clarified the text

**Pull request merged**